akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1452,6 +1452,20 @@ trait FlowOps[+Out, +Mat] {
   def idleTimeout(timeout: FiniteDuration): Repr[Out] = via(new Timers.Idle[Out](timeout))
 
   /**
+    * If the time between emission of element and following backpressure notification exceed the provided timeout,
```

Comment:
I think `IdeInject` (aka `keepAlive`) combined with map is capable of producing same end effect. Does it mean we don't want to introduce this stage (similar situation to #19422) or maybe we want `flow.keepAlive.map` wrapped as `backpressureTimeout`?